iml-agent-comms/src/main.rs

```diff
@@ -128,13 +128,17 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let shared_hosts2 = Arc::clone(&shared_hosts);
     let shared_hosts3 = Arc::clone(&shared_hosts);
 
+    let pool = iml_rabbit::connect_to_rabbit(2);
```

Comment:
2?

Reply:
This is the maximum number of connections that can be in the pool.

Each service has different requirements for connections.

Reply:
Ah, I see why 2.